asserts/snap_asserts.go

```diff
@@ -171,16 +172,16 @@ func (snaprev *SnapRevision) SnapDigest() string {
 	return snaprev.Header("snap-digest")
 }
 
+// SnapSize returns the size in bytes of the snap submitted to the store.
+func (snaprev *SnapRevision) SnapSize() uint64 {
+	return snaprev.snapSize
+}
+
 // SnapRevision returns the revision of the snap-id assigned to this build.
```

Comment:
unrelated to this branch, but this comment is clunky/ambiguous, can we improve it?

SnapRevision returns the revision assigned to this build of snap-id ?